client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -45,22 +52,22 @@ class MappedDomainType extends React.Component {
 			return null;
 		}
 
-		const learnMoreLink = ( linksTo ) => (
-			<a href={ linksTo } target="_blank" rel="noopener noreferrer" />
-		);
+		let setupInstructionsMessage;
 		let primaryMessage;
 		let secondaryMessage;
 
 		if ( isSubdomain( domain.name ) ) {
+			setupInstructionsMessage = translate(
+				'Follow these instructions to set up your subdomain mapping:'
```

Comment:
To be consistent, let's go with `'You need to follow these instructions to finish connecting the %(domainName)s subdomain to your WordPress.com site:',` here as well.